WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -379,7 +382,14 @@ private async Task StepTransactionSigningPhaseAsync(CancellationToken cancellati
 				else if (round.TransactionSigningTimeFrame.HasExpired)
 				{
 					round.LogWarning($"Signing phase failed with timed out after {round.TransactionSigningTimeFrame.Duration.TotalSeconds} seconds.");
-					await FailTransactionSigningPhaseAsync(round, cancellationToken).ConfigureAwait(false);
+					if (round.FastSigningPhase)
```

Comment:
I think we can remove the boolean flag from `Round.cs`, if here we check for this

```suggestion
					if (round.Alices.Any(a => !a.ReadyToSign))
```

WDYT?

Reply:
I prefer to be explicit because:
 * The identifier `FastSigningPhase` itself serves as documentation.
 * When you decide to change the condition `round.Alices.Any(a => !a.ReadyToSign))` in the future, you can change it in one place.

However, I have no problem with what you suggested because this is closer to the style that is used in your repository.